lib/matplotlib/tests/test_lines.py

```diff
@@ -199,6 +199,30 @@ def test_step_markers(fig_test, fig_ref):
     fig_ref.subplots().plot([0, 0, 1], [0, 1, 1], "-o", markevery=[0, 2])
 
 
+@check_figures_equal(extensions=('png',))
+def test_markevery(fig_test, fig_ref):
+    np.random.seed(42)
+    t = np.linspace(0, 3, 14)
+    y = np.random.rand(len(t))
```

Comment:
Optional: There is no gain from random numbers here. You could just use `y = t**2` or any other simple function.